package.json

```diff
@@ -28,16 +28,20 @@
     "eslint-plugin-import": "2.2.0",
     "eslint-plugin-jsx-a11y": "3.0.2",
     "eslint-plugin-react": "6.9.0",
+    "extract-text-webpack-plugin": "^2.0.0",
```

Comment:
let's keep dependencies to fixed versions. We don't want surprises when they update :)